Cargo.toml

```diff
@@ -56,13 +57,16 @@ version = "0.1.56"
 edition = "2021"
 
 [workspace.lints.rust]
-unexpected_cfgs = { level = "warn", check-cfg = ['cfg(coverage_nightly)'] }
+unexpected_cfgs = { level = "warn", check-cfg = [
+    'cfg(legacy_builder_refactored, values(none()))',
```

Comment:
This seems to be inconsistent with `legacy_refactored` in the `yml` files. Shouldn't they use the same `cfg`?

Reply:
Right, thanks for the catch!